ui/src/components/core/input/input-file.tsx

```diff
@@ -0,0 +1,79 @@
+import { Flex } from '../../layout';
+import { dripStitches } from '../../../theme/stitches';
+import { useRef, useState } from 'react';
+import { Icon } from '../icon';
+import { StyledErrorMessage } from './input-error-message';
+
+const { styled } = dripStitches;
+
+const BorderInput = styled('div', {
+  borderStyle: 'solid',
+  borderColor: '$gray7',
+  width: '$22',
+  height: '$22',
+  transition: 'border-color 0.2s ease-in-out',
+  borderWidth: '$default',
+  borderRadius: '$lg',
+  zIndex: '$docked',
+  my: '$1h',
+
+  '&:hover': {
+    borderColor: '$gray8',
+  },
+});
+
+const validateFileSize = (file: File) => {
+  const fileSize = file.size / 1024; // in KB
+  return fileSize <= 10;
+};
+
+export const InputFileStyled = () => {
+  const inputFileRef = useRef<HTMLInputElement>(null);
+  const [file, setFile] = useState<File | null>(null); //TODO: do it with redux

```

Comment:
I think this component would be cool to have props like `value` and `onChange` so it turns out to be controlled like a common input element when we use it on parent components. It seems like we could pass props to the Flex wrapper, so on a parent element we can also control the styling to for example fill a row in a form. It seems like passing the state controlling role to the parent is more correct than handling directly to Redux, because in that way we can for an example have multiple file inputs in a form.